src/components/IvanovDS/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./IvanovDSController.js');
```

Comment:
Используем синтаксис ES6
```suggestion
export { default } from './IvanovDSController';;
```